packages/mongodb-chatbot-server/src/routes/conversations/addMessageToConversation.ts

```diff
@@ -44,6 +44,7 @@ export const DEFAULT_MAX_USER_MESSAGES_IN_CONVERSATION = 7; // magic number for
 export type AddMessageRequestBody = z.infer<typeof AddMessageRequestBody>;
 export const AddMessageRequestBody = z.object({
   message: z.string(),
+  clientContext: z.object({}).passthrough().optional(),
```

Comment:
what does `.passthrough()` do here?

Reply:
Passthrough is basically equivalent to `additionalProperties: true` in JSON schema. If you don't have it then zod parsing strips out any fields that aren't in the schema. Might be unnecessary here if we go the custom schema route you suggested.